whatsapp_fn.py

```diff
@@ -102,6 +107,7 @@ def Get_Message():
         if records_a.shape[0]>0:
             hist = 'v'
             name = records_a.loc[0,'name']
+            add_message(frm, frm, bot_number, text, "text", "whatsapp", "incoming")
```

Comment:
Same as above @Mohit177

Reply:
@satishck1992 as discusses over call I am marking this comment as not relevant now.